types/cloudflare-apps/cloudflare-apps-tests.ts

```diff
@@ -0,0 +1,80 @@
+import * as cloudflareApps from "cloudflare-apps";
```

Comment:
No, but how should we load the types into a particular file then?
Because it does not add them automatically, probably to not pollute the namespace.
Just like React adds `JSX.Element` (well, the whole `JSX` namespace).

If you have better way to solve this, we're glad to hear 👍